cli/src/commands/init.rs

```diff
@@ -23,28 +27,52 @@ pub async fn handle_init(api: &mut PhylumApi, matches: &ArgMatches) -> CommandRe
         if !should_continue {
             return Ok(ExitCode::ProjectAlreadyInitialized.into());
         }
+
+        println!();
     }
 
     let cli_project = matches.get_one::<String>("project");
     let cli_group = matches.get_one::<String>("group");
 
     // Interactively prompt for missing information.
-    let (project, group) = prompt(cli_project, cli_group)?;
+    let (project, group) = prompt_project(cli_project, cli_group)?;
 
     // Attempt to create the project.
-    let response = project::create_project(api, &project, group.clone()).await?;
+    let result = project::create_project(api, &project, group.clone()).await;
 
     // If project already exists, just link to it.
-    match response {
-        CommandValue::Code(ExitCode::AlreadyExists) => {
-            project::link_project(api, &project, group).await
+    let mut project_config = match result {
+        Err(PhylumApiError::Response(ResponseError { code: StatusCode::CONFLICT, .. })) => {
+            print_user_success!("Successfully linked to project {project:?}");
+            project::link_project(api, &project, group).await.context("Unable to link project")?
```

Comment:
```suggestion
            project::link_project(api, &project, group).await.context("Unable to link project")?
            print_user_success!("Successfully linked to project {project:?}");
```

It's probably a good idea to swap the order of these lines to avoid the possibility of:
```
✅ Successfully linked to project foo
❗ Unable to link to project ...
```